### DISCUSSION TITLE
PalBATCH algorithm

### DISCUSSION THREAD

**Discussion by Suesmith12** [Ergo-Tools/ActiPASS#87] [Help and Q&A (processing, quality checking or statistics)] [answered]
Hi Pasan,

We are using ActivPAL 4s in our trial. When converting from datx files to csv files (for subsequent processing with ActiPASS), which classification algorithm is best to select in PALBatch please? I think these options have been updated more recently.

Thank you.
Sue

![Screenshot 2024-05-08 160719](https://github.com/Ergo-Tools/ActiPASS/assets/148974338/357e4791-60e7-4bae-97c8-6c8b621252b9)

**Comment by pasan-h** [comment, marked as answer]
Hi Sue,
Most probably it doesn't matter. Because I assume those settings are related to other types of exports with classifications. I think all settings which affects raw accelerometer data are given under "Sensor data exports"  section. However, it's worth asking this question from ActivPAL support. For an example it's theoretically possible that exported raw accelerometer data are orientation-corrected and that depends on the selected algorithm. But I doubt it. 

The recommended settings are as shown in the screenshot below. 
![image](https://github.com/Ergo-Tools/ActiPASS/assets/26480941/d5100f6c-2152-4e1f-b8b5-955d734535c5)

Using the 'uncompressed' option makes the CSV files larger. However we have not tested any implications between the 'compressed' and 'uncompressed' CSV formats. ActiPASS supports both formats. At least in theory using the 'compressed' format can influence non-wear and sleep algorithms (perhaps very minimal and practically it doesn't matter). Anyway that's why I recommend using the 'uncompressed' format.

The "g-format" and "integer format" does not make any difference and we can use the "integer-format" which produces smaller CSV files. Since automatic device calibration is built-in to ActiPASS and it's always on, we can let ActiPASS handle device calibration and choose "uncalibrated" option.
Cheers!
/Pasan

**Reply by Suesmith12** [reply]
Hi Pasan,

As always, thank you for your swift reply and for your help - much appreciated.
Thanks,
Sue
